Jenkinsfiles/utils.groovy

```diff
@@ -55,34 +55,38 @@ def notify_irc(Map args) {
     sh command
 }
 
-def make(cmd, cmd_display) {
-    def target = get_target()
-    def tag = get_commit_tag()
-    def nick = "mdn${target}push"
-    def repo_upper = get_repo_name().toUpperCase()
+def sh_with_notify(cmd, display, notify_on_success=false) {
+    def nick = "mdn-${env.BRANCH_NAME}"
```

Comment:
does the nick change with every branch on purpose?

Reply:
Yes, I thought it'd be a quick indicator of where the message is coming from (although the branch name is also in the message). It's helpful for example when we're doing a push of both kuma and kumascript at the same time, since they get different nicks.

I'm totally open to a different approach though!

Reply:
It does make my IRC a bit wordy:

![screenshot 2017-11-16 16 10 14](https://user-images.githubusercontent.com/286017/32918661-b6bda454-cae8-11e7-9fe4-ea41c21748a6.png)

Let's keep this for a week or so and a few pushes. I might have a stronger opinion then.